armi/cases/case.py

```diff
@@ -425,6 +426,33 @@ def _endCoverage(cov=None):
             combinedCoverage.html_report()
             combinedCoverage.xml_report()
 
+    @staticmethod
+    def _getCoverageRcFile(makeCopy=False):
+        """Helper to provide the coverage configuration file according to the OS.
+
+        Parameters
+        ----------
+        makeCopy : bool (optional)
+            Whether or not to copy the coverage config file to an alternate file path
+
+        Returns
+        -------
+        covFile : str
+            path of coveragerc file
+        """
+        covRcDir = os.path.abspath(context.PROJECT_ROOT)
+        if platform.system() == "Windows":
+            if makeCopy == True:
+                # Make a copy of the file without the dot in the name
+                shutil.copy(
+                    os.path.join(covRcDir, ".coveragerc"),
+                    os.path.join(covRcDir, "coveragerc"),
+                )
+            covFile = os.path.join(covRcDir, "coveragerc")
+        else:
+            covFile = os.path.join(covRcDir, ".coveragerc")
```

Comment:
It looks to me like you could make this simpler:

```python
if platform...
    if makeCopy...
        # Make ...

covFile = os.path.join(covRcDir, ".coveragerc")
```

It just _looks_ like that `else` is unnecessary, and you could have the `covFile =` line once instead of twice.  Right?